### DISCUSSION TITLE
How do you set up the `Regression Detector`?

### DISCUSSION THREAD

**Discussion by hirehamir** [vectordotdev/vector#21489] [General]
The automated `Regression Detector Results` that show up on a pull request are really interesting. How was it set up?

**Comment by hirehamir** [comment]
https://github.com/vectordotdev/vector/pull/21479#issuecomment-2407996561 is an example of the results.

**Reply by jszwedko** [reply]
https://github.com/vectordotdev/vector/blob/master/.github/workflows/regression.yml is the workflow. It relies on some open source tools (like [lading](https://github.com/datadog/lading)), but also a service internally hosted at Datadog that runs the benchmarks and does the statistical analysis.
